openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -6,7 +6,7 @@ import '../../../../vendor/js/jquery-ui/jquery-ui-1.12.1.min.js';
 import '../../../../vendor/js/colorbox/1.5.14.js';
 // jquery.form#2.36 not on npm, no longer getting worked on
 import '../../../../vendor/js/jquery-form/jquery.form.js';
-// jquery-validate#1.6 not on npm
+// jquery-validate#1.19.1 not on npm
```

Comment:
it's on npm so this comment is incorrect. Let's replace this with import 'jquery-validation'

Reply:
Will do.